schema/paramset.json

```diff
@@ -45,13 +59,19 @@
                 },
                 "msg": {
                     "type": "integer",
-                    "minimum": 1
+                    "minimum": 1,
+                    "$comment": "For KEMs, size of the shared secret. #TODO signatures?"
```

Comment:
Signature size for signatures

Reply:
I thought so, but I was not sure about the details. (E.g. Do they always include a hash function internally, which would mean that the message size is practically illimited?) It'll probably clear up once I've added one or two signature schemes...

Reply:
I would say this holds the size of the internally hashed value which is then signed. The signature size is stored in `ct/sig`.

Reply:
That's probably the only value that is significant... We'll see. I'll remove the `TODO` once I've had a look at a few sig schemes if nobody else has removed it before :)